recipes/iiboost/build.sh

```diff
@@ -0,0 +1,19 @@
+
+#!/bin/bash
+
+if [[ "$(uname)" == "Darwin" ]]; then
+    export ENABLE_OPENMP=no
+else
+    export ENABLE_OPENMP=yes
+fi
+
+mkdir build && cd build
+cmake \
+    -DCMAKE_PREFIX_PATH=$PREFIX \
+    -DITK_DIR=$PREFIX/lib/cmake/ITK-4.11/ \
+    -DIIBOOST_USE_OPENMP=$ENABLE_OPENMP \
+	..
+
+make
```

Comment:
It might need to be set wherever `pickle` is imported. Initially I was looking for an environment variable for this, but didn't find one. Maybe [this]( https://stackoverflow.com/questions/28218466/unpickling-a-python-2-object-with-python-3/28218598#28218598 ) is more useful. Would raise this point for discussion in the Python 3 PR.